src/rgw/rgw_civetweb.h

```diff
@@ -25,6 +25,7 @@ class RGWMongoose : public RGWStreamIO
   bool has_content_length;
   bool explicit_keepalive;
   bool explicit_conn_close;
+  bool no_more_data_to_read;
```

Comment:
no_? I think it is more natural to save the non-negated version; or to rename this something non-negated (e.g., eod)